aten/src/ATen/BatchingRegistrations.cpp

```diff
@@ -340,7 +340,12 @@ Tensor select_backward_batching_rule(const Tensor& grad, IntArrayRef input_sizes
   return grad_physical.newLogicalFromPhysical(grad_input);
 }
 
-Tensor slice_batching_rule(const Tensor& self, int64_t dim, int64_t start, int64_t end, int64_t step) {
+Tensor slice_batching_rule(
```

Comment:
cc: @zou3519 , i think these changes are harmless but double checking

Reply:
Yup, this lgtm